libcst/codemod/visitors/tests/test_add_imports.py

```diff
@@ -481,11 +481,11 @@ def bar() -> int:
                 return 5
         """
 
-        self.assertCodemod(before, after, [("a.b.c", "D", None)])
+        self.assertCodemod(before, after, [Import("a.b.c", "D", None)])
 
     def test_add_import_preserve_doctring_multiples(self) -> None:
         """
-        Should preserve any doctring if adding to the beginning.
+        Should preserve any doctring if addImport(ng to the beginning.
```

Comment:
![](https://media1.giphy.com/media/1X7lCRp8iE0yrdZvwd/giphy.gif?cid=ecf05e471zq5pw04j0512i08lvyfa1qhautsn6v4gddd9lzl&rid=giphy.gif&ct=g)

Reply:
hit the wrong key in vim and didn't notice (: